backend/src/main/java/com/bakdata/conquery/models/events/stores/specific/DateRangeTypeCompound.java

```diff
@@ -0,0 +1,128 @@
+package com.bakdata.conquery.models.events.stores.specific;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+import com.bakdata.conquery.io.cps.CPSType;
+import com.bakdata.conquery.models.common.daterange.CDateRange;
+import com.bakdata.conquery.models.events.Bucket;
+import com.bakdata.conquery.models.events.stores.primitive.IntegerDateStore;
+import com.bakdata.conquery.models.events.stores.root.ColumnStore;
+import com.bakdata.conquery.models.events.stores.root.DateRangeStore;
+import com.bakdata.conquery.models.events.stores.root.DateStore;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+import lombok.ToString;
+
+/**
+ * Stores {@link CDateRange} by storing only the references of the start and end of the {@link CDateRange}.
```

Comment:
Äh, da fehlt was glaube ich, du referenzierst ja die Stores und nicht die CDateRanges

Reply:
ja ein DateRange ist in unserem durch einen Start und ein Ende gekennzeichnet. So habe ich es in meinem Satz gemeint